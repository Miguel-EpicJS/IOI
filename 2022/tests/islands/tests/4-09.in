4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
5 12
3 2
3 2
1 4
1 4
2 3
2 3
0 1
0 1
4 3
4 3
4 3
4 3
