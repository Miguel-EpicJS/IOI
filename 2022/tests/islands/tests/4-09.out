91041670623a69783795ed82bda33426101ce311
OK
1
16
6 2 8 0 4 8 2 6 7 2 8 4 0 8 2 7
